spec/features/users/user_profile_spec.rb

```diff
@@ -1,6 +1,29 @@
 require 'rails_helper'
 
 feature 'User profile' do
+  context 'account status badges' do
+    before do
+      sign_in_live_with_2fa(profile.user)
+    end
+
+    context 'LOA1 account' do
+      let(:profile) { create(:profile) }
+
+      it 'shows a "Basic Account" badge with a tooltip' do
+        expect(page).to have_content(t('headings.profile.basic_account'))
```

Comment:
I removed the feature specs and used view specs instead (I thought the request was to have one instead of the other). I'm going to go with my gut here and revert that change to a view spec, and use a feature spec instead since I believe it will be a less-fragile test.